### DISCUSSION TITLE
differentiation of composite functions

### DISCUSSION THREAD

**Discussion by 02y16** [sympy/sympy#26586] [Q&A]
Hi SymPy team, I am trying to use SymPy to derive mathematical formulas
Can Sympy currently achieve the following functions:    
replace the derivative of u over x with u'
for example, 
U=2*x
V=x**2   
F=u+v+x 
The result of the derivative obtained is F_prime =u '+v' + 1 instead of 2*x+3

**Comment by louis-chevallier** [comment]
```
import sympy
x = sympy.symbols("x")
u, v = 2 * x, x**2
sympy.diff(u + v + x, x)
```
actually produces : $2 x + 3$
